scripts/new_project.kts

```diff
@@ -103,6 +107,32 @@ object NewProject {
             }
     }
 
+    private fun renameApplicationClass() {
+        showMessage("=> 🔎 Renaming application class...")
+        File(projectPath)
+            ?.walk()
+            .find { it.name == "$TEMPLATE_APPLICATION_CLASS_NAME.kt" }
+            ?.let { templateApplicationFile ->
+                val newApplicationPath = templateApplicationFile.absolutePath.replaceAfterLast(
+                    delimiter = fileSeparator,
+                    replacement = "$applicationClassName.kt"
+                )
+                val newApplicationFile = File(newApplicationPath)
+                templateApplicationFile.renameTo(newApplicationFile)
+            }
+
+        File(projectPath)
+            ?.walk()
+            .filter { it.name.endsWith(".kt") || it.name.endsWith(".xml") }
```

Comment:
@Tuubz can we specific `AndroidManifest.xml` file like above? 🙏  is there any other `.kt` files I didn't notice here? 😁

Reply:
Aaah, good question 👍 

There's `AndroidManifest.xml` but also the application class itself:

![Screen Shot 2565-05-20 at 17 39 01](https://user-images.githubusercontent.com/18277915/169511063-31867fa8-fd20-430b-b646-49fbe2398cd8.png)

Do you think we should change it? 🙏

Reply:
ah ha, it's good now. Thanks for explanation 🌮

Reply:
@Tuubz Which file of `.kt` that we have to rename here 🙏